mail_restrict_follower_selection/models/mail_wizard_invite.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015 Therp BV <http://therp.nl>
+# Copyright (C) 2017 Komit <http://www.komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from lxml import etree
+
+from odoo import api, models
+from odoo.osv import expression
+from odoo.tools.safe_eval import safe_eval
+
+
+class MailWizardInvite(models.TransientModel):
+    _inherit = "mail.wizard.invite"
+
+    @api.model
+    def _mail_restrict_follower_selection_get_domain(self, res_model=None):
+        if not res_model:
+            res_model = self.env.context.get("default_res_model")
+        parameter_name = "mail_restrict_follower_selection.domain"
+        parameter_domain = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param(
+                "{}.{}".format(parameter_name, res_model),
+                self.env["ir.config_parameter"]
+                .sudo()
+                .get_param(parameter_name, default="[]"),
+            )
+        )
+        domain = expression.AND(
+            [safe_eval(parameter_domain), self._fields["partner_ids"].domain]
+        )
+        return domain
+
+    @api.model
+    def get_view(self, view_id=None, view_type="form", **options):
```

Comment:
This get_view method is not called in every instance, so the domain for the partner_ids field is not set correctly.

get_view is only called in the first instance, after that, onchange method is called, even after changing models.

If you add a follower in one model, then go to another model with a different domain set and add a follower again, the get_view function is not called, leading to an incorrect domain set on partner_ids.

Hope this is clear.